components/item/ItemHero.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <AwesomeSwiper
+      :europeana-identifier="id"
+      :media="media"
+      @select="selectMedia"
+    />
+    <b-container class="d-flex justify-content-around mt-5">
+      <RightsStatementButton
+        v-if="rightsStatementIsUrl"
+        :rights-statement="rightsStatement"
+      />
+      <DownloadButton
+        :url="downloadUrl"
+        :disabled="disabled"
+        :target="target"
+      />
+      <ShareButton />
+      <SocialShareModal :media-url="mediaUrl" />
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import AwesomeSwiper from './AwesomeSwiper';
+  import DownloadButton from '../generic/DownloadButton.vue';
+  import RightsStatementButton from '../generic/RightsStatementButton.vue';
+  import SocialShareModal from '../sharing/SocialShareModal.vue';
+  import ShareButton from '../sharing/ShareButton.vue';
+
+  export default {
+    components: {
+      AwesomeSwiper,
+      DownloadButton,
+      RightsStatementButton,
+      SocialShareModal,
+      ShareButton
+    },
+    props: {
+      media: {
+        type: Array,
+        default: () => []
+      }
+    },
+    data() {
+      return {
+        mediaUrl: 'https://www.example.org/media',
+        id: '/2020601/https___1914_1918_europeana_eu_contributions_10265',
+        rightsStatementIsUrl: true,
+        disabled: false,
```

Comment:
I'd say get rid of the `disabled` property for now. We'll need to add it again later, but there's additional work here for doing this lookup properly and then also adding another notice to the page. For now it doesn't do anything and when it's added with functionality it should be named something more descriptive.

Reply:
yes good point